users/views.py

```diff
@@ -105,4 +105,29 @@ def post(self,request):
                 return JsonResponse({'message':'SIGNIN_SUCCESS', 'TOKEN':token.decode()}, status=200)
             return JsonResponse({'message':'WRONG_PASSWORD'}, status=401)
         except KeyError:
-            return JsonResponse({'message': 'KEY_ERROR'}, status=401)
\ No newline at end of file
+            return JsonResponse({'message': 'KEY_ERROR'}, status=401)
+
+class KakaoView(View):
+    def post(self, request): 
+        try:
+            access_token = request.headers['Authorization']
+            headers      = ({'Authorization' : f'Bearer {access_token}'})
+            url          = 'https://kapi.kakao.com/v2/user/me'
+            data         = requests.get(url, headers=headers).json()
+
+            if not User.objects.filter(social_id = data['id']).exists():
+                new_library = Library.objects.create(name=f'{data["properties"]["nickname"]}의 서재')
+                usertype    = UserType.objects.get(name='kakao')
+                new_user    = User.objects.create(
+                    social_id   = data['id'],
+                    nickname    = data['properties']['nickname'],
+                    library_id  = new_library.id,
+                    usertype_id = usertype.id
```

Comment:
```suggestion
                    library  = new_library,
                    usertype = usertype
```
큰 차이는 아니지만 이렇게 해볼 수 도 있습니다~